electrum/commands.py

```diff
@@ -1171,6 +1171,7 @@ def add_global_options(parser):
 
 def add_wallet_option(parser):
     parser.add_argument("-w", "--wallet", dest="wallet_path", help="wallet path")
+    parser.add_argument("-f", "--forgetconfig", action="store_true", dest="forget_config", default=False, help="Forget config on exit")
```

Comment:
`-f` is already taken to specify the `fee` arg for relevant commands
Especially for "niche" options, there does not need to be a single char shortcut at all.

Reply:
Right. Removed and pushed.